lib/Connectable.php

```diff
@@ -0,0 +1,9 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-y5zk7w.php, but found phpcs-scan-Y5zk7w.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).